### ISSUE TITLE
Latest update - Font legibility

### ISSUE THREAD

**Issue by tomhorn3r** [itfoundry/Poppins#13]
The latest update of the font Poppins seems to have reduced the legibility of certain characters, specifically the lowercase 'e' (amongst others)

This is demonstrated clearly in the before/after gif submitted to Google Fonts (see below)

![29409663-34156072-8345-11e7-99d6-bbb319cbe8ea](https://user-images.githubusercontent.com/31278905/29615673-b12c07f8-8806-11e7-9e14-fdb4c1ff94c0.gif)

**Comment by wegomusic** [comment]
Totally right. Latest changes ruined the typeface. For example dot character now it's squared, 'e' character also has changed. Font feeling has changed. It's specially notable using SemiBold version. Now I'm searching an alternative Google Font.

**Comment by tomhorn3r** [comment]
Likewise @wegomusic :(

Can anything be done to correct this, @lianghai ?

**Comment by SlavVitanov** [comment]
Characters like the lowercase a, b, d, e and m are not rendering properly since the last update. 

To be specific the parts of the characters where the bowl (the rounded shape) is flowing into the stem gets extremely thinned out in certain cases - for example if you switch Google Slides into full screen mode. 

![screen shot 2017-09-06 at 10 39 45](https://user-images.githubusercontent.com/31689790/30105448-85005e3c-92f0-11e7-84ce-8a5962bb3a10.jpg)

This is causing us problems as we work predominantly in Google Slides and our whole branding is based on Poppins.

**Comment by davelab6** [comment]
here is docs present mode:

<img width="656" alt="screen shot 2017-09-06 at 18 21 03" src="https://user-images.githubusercontent.com/261579/30125402-358f208c-9330-11e7-8438-756b382c915f.png">

and here is docs edit mode:

<img width="499" alt="screen shot 2017-09-06 at 18 21 08" src="https://user-images.githubusercontent.com/261579/30125414-40200b9c-9330-11e7-9789-c3c03e7c05dc.png">

Indeed, it does look very different.

**Comment by SlavVitanov** [comment]
I don't think the rendering problem is limited to Google Slides only. 

It's happening in Google Fonts too.

<img width="1172" alt="screen shot 2017-09-07 at 10 17 57" src="https://user-images.githubusercontent.com/31689790/30155984-4441ab82-93b6-11e7-89bc-9c7f94d35049.png">

**Comment by kustard** [comment]
As a regular user of the font in Google, I agree with all the above and would welcome a fix

**Comment by tderkert** [comment]
I get a feeling that there are necessary in-between states missing for some glyphs, used for some visual adjustments. Now /e for example interpolates linearly from Thin to Black which doesn't feel right.

**Comment by hornergraphic** [comment]
This font rendering is a real issue for our brand, please fix this soon!

**Comment by jchristopher88** [comment]
Hoping there is a fix for this soon as we rely on Poppins for a platform that many people use on a daily basis.

**Comment by davelab6** [comment]
Because the update introduced new styles, I can't roll back the family
without breaking their usage of the new weighs. satya has promised to make
an update that fixes this, however I guess it won't reach gf production
servers until October. Until then you can obtain the previous release and
self host it

**Comment by satyarajpurohit** [comment]
Yes, we're working on a fix but it may take some time. In the meantime, please use the older files. Thanks!

**Comment by satyarajpurohit** [comment]
Hello everyone! Meanwhile @davelab6 update these files on GoogleFonts, you can download the updated fonts here: https://github.com/itfoundry/Poppins/releases/tag/v3.100

Thanks @lianghai.

**Comment by davelab6** [comment]
Thank you Satya and Liang!!!! Super appreciate it :)

**Comment by tomhorn3r** [comment]
Thanks so much guys, really appreciate it.

Looking forward to seeing the update on Google Fonts!

**Comment by tomhorn3r** [comment]
Hey everyone,

Any idea when we are likely to see this update in Google Fonts?

Tom

**Comment by davelab6** [comment]
@m4rc1e is helping get fonts updated upstream into Google Fonts, and this
is high on our list :)

**Comment by wilkinsocks** [comment]
Thanks for the update, Dave

**Comment by seansusu** [comment]
Its a gorgeous font but every web-graphics editor such as Vectr, Gravit, and Figma are all showing the rendering problems thanks to Google fonts. Quite depressing. Thanks for the old fonts. Will be doing fixes.

**Comment by wilkinsocks** [comment]
Hey @davelab6 / @m4rc1e, just wondering if the latest version of this has been pushed to Google Fonts?

The 'e' on the Google Fonts page looks like it still has the rendering issue: https://fonts.google.com/specimen/Poppins

Thanks for your help in advance!

**Comment by scottbuscemi** [comment]
This issue still persists...

**Comment by oscaralexander** [comment]
+1 to all of the above. Whenever I see Poppins looking nice on a website, it’s not the Google Fonts version. I would love to see it reverted as readability and general aesthetic have really suffered.

That said, I’m getting the impression the foundry isn’t really bothering with this issue/thread 😔

**Comment by Arilas** [comment]
![image](https://user-images.githubusercontent.com/3540524/53213100-4be39380-3650-11e9-9ef2-929b3559a16e.png)

For some reason, the version from Google Fonts has a bug with `e` letter on 18px font size on screens without high-dpi.  This issue can also be seen on Google Fons website (change size to 18).

**Comment by scottbuscemi** [comment]
![screen shot 2019-02-21 at 5 24 54 pm](https://user-images.githubusercontent.com/4336848/53213394-9ee02b00-35fd-11e9-8b0b-e968c7af8312.png)
You think that's bad? Check out the two different kinds of lower case L's ;)

**Comment by WesCooper** [comment]
Is there any ETA on when this will be fixed? I really don't want to be using font files to get the original Poppins font and the Google version has some serious legibility issues 😔

**Comment by scottbuscemi** [comment]
From @lianghai on Twitter:

> Mmm, it seems the proposed update got stuck here https://github.com/itfoundry/Poppins/commit/8ead97608682a8ac08a5bce64266852cd50a5537 …—I’ll confirm with Google Fonts people and see what I can do…

https://twitter.com/lianghai/status/1101693244520333314

**Comment by JohannesBooy** [comment]
+1 - watching with interest.

**Comment by m4rc1e** [comment]
Just reviewed v4.002 agains the version we host on GF. See #30.

**Comment by scottbuscemi** [comment]
@m4rc1e Did the update make it to GF? I definitely understand the concerns about scale, but the strange aliasing on smaller font sizes is concerning.

**Comment by wilkinsocks** [comment]
@scottbuscemi It looks like Google's QA flagged up a few other issues with the latest version; although seems @lianghai fixed them now 🙌

**Comment by WesCooper** [comment]
Awesome 🙌Looking forward to being able to use this again! Thanks for updates and progress!

**Comment by scottbuscemi** [comment]
> ![screen shot 2019-02-21 at 5 24 54 pm](https://user-images.githubusercontent.com/4336848/53213394-9ee02b00-35fd-11e9-8b0b-e968c7af8312.png)
> You think that's bad? Check out the two different kinds of lower case L's ;)

My issue seems to be resolved now - the font from Google Fonts is beautiful again.

**Comment by satyarajpurohit** [comment]
Hi @m4rc1e, 
what is stopping you now from pushing these fonts to GoogleFonts? 

(cc @davelab6, @lianghai)

**Comment by m4rc1e** [comment]
@satyarajpurohit I have concerns about the horizontal linespace, https://github.com/itfoundry/Poppins/issues/30. It differs greatly from what we serve. I'll push it if @davelab6 is happy.

**Comment by satyarajpurohit** [comment]
Hi @m4rc1e,

The last version has some serious issues and we'd like to fix it with this update. Several people are already using these fonts from our Github and requested that we push it on GoogleFonts. This might cause inconvenience to some but most will be happy to see this version live. 

Hi @davelab6, can you please confirm?

**Comment by davelab6** [comment]
https://github.com/itfoundry/Poppins/issues/30#issuecomment-470981011 is not replied to

**Comment by davelab6** [comment]
The horizontal line layout spacing change is fine as long as the Metrics and Kerning are overall better and don't introduce bugs. So I think this is a good candidate to at least get to the GF sandbox.

**Comment by satyarajpurohit** [comment]
Thank you, @davelab6! 
cc @m4rc1e

**Comment by wilkinsocks** [comment]
@davelab6 Do you know when this will make it's way to Google Font's production?

**Comment by majlan** [comment]
April 2020 and 'e' is still being rendered wrong. Is anything going to happen?

**Comment by oscaralexander** [comment]
It seems to be fixed! 👏🏻
![image](https://user-images.githubusercontent.com/1172527/89174501-3fbe6e00-d586-11ea-95c2-4fce7dbe3886.png)

**Comment by marijoo** [comment]
It seems to be fixed if you take a look at the Google Fonts website but on my mac the **e** still looks awful.
